com/williamfiset/datastructures/binarysearchtree/SplayTree.java

```diff
@@ -0,0 +1,274 @@
+import com.williamfiset.datastructures.utils.TreePrinter;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+
```

Comment:
Can you also add a section in the README that points to this file.

Reply:
Done